indexer/services/socks/__tests__/lib/message-forwarder.test.ts

```diff
@@ -182,6 +182,8 @@ describe('message-forwarder', () => {
   };
 
   beforeAll(async () => {
+    config.BATCH_PROCESSING_ENABLED = false;
```

Comment:
nit: Shouldn't tests pass with batch processing enabled?

Reply:
The ordering of functions in this test is not exactly like `index.ts`. `startConsumer` gets called in beforeAll, while the `messageForwarder` is initialized in each test. It's a bit of a pain to fix. I just set to false for now. If you feel strongly, I can add a TODO to fix this test.